Jenkinsfile

```diff
@@ -131,7 +131,9 @@ pipeline {
       }
       steps {
         script {
-          maven cmd: "deploy -Dmaven.test.skip=true -Divy.engine.version='[10.0.0,]'"
+          maven cmd: "deploy -Dmaven.test.skip=true " +
+                  "-Divy.engine.version='[10.0.0,]'" +
```

Comment:
maven range is wrong? for 10 branch. i thought i've fixed it?!

Reply:
@alexsuter 🤔 https://github.com/axonivy/dev-workflow-ui/commit/3deaa8da76417827e1d2c289ea6e753274c00947

Reply:
ah sorry...

Reply:
@ivy-lli seems that you adapt the wrong maven version range. this is currently integrating with 11.1 instead of 10.0